rtdata/languages/default

```diff
@@ -3126,7 +3126,7 @@ TP_LOCALLAB_THRES;Threshold structure
 TP_LOCALLAB_THRESDELTAE;ΔE scope threshold 
 TP_LOCALLAB_THRESRETI;Threshold
 TP_LOCALLAB_THRESWAV;Balance threshold
-TP_LOCALLAB_TLABEL;TM Data Min=%1 Max=%2 Mean=%3 Sigma=%4 (Threshold)
+TP_LOCALLAB_TLABEL;TM Data Min=%1 Max=%2 Mea=%3 Sig=%4(thr)
```

Comment:
"Mea" is not a standard abbreviation for anything. If trying to shorten stuff, why not use standard abbreviations ("AM" for "arithmetic mean") or symbols (Σσς). While I am against RT's long tooltips, _this_ is exactly what tooltips are for - to explain that e.g. "AM" means "arithmetic mean".

Reply:
I played with the advanced retinex tools:
https://youtu.be/T7_ZAc-27dE
I see several issues here:
* Updates to that label seem to occur randomly. I had to move one point up and down several times to get a number change.
* The magnitude of the numbers changes, this moves the different numbers around in the label, making it impossible to track changes of of one particular value.
* Occasionally it gets bigger than the panel width, GTK adds scrollbars which moves the GUI the user interacts with (I couldn't check with recent changes)
* Sometimes MIN is bigger than MAX and MEAN outside [MIN,MAX] - but perhaps that's normal?

Question: Is it really important to give the user exact numbers? Couldn't this be presented simply with a horizontal bar showing two or three marks? color them e.g. min=black, max=white, mean=gray? Or might it even be possible to visualize those values in the transition map graph the user interacts with?